tools/test_containers_compose

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+for i in webui; do
+  (
+    cd container/$i/ && \
+    sudo docker-compose build -q --parallel && \
+    sudo docker-compose up -d && \
+    sleep 30 && \
```

Comment:
As usual I have my problems with approving something that introduces sleep with more than 1 second :)

Reply:
Ok, I check if is possible to remove that

Reply:
But the reason for that is because maybe we have an Up in our containers in the the second 1 but not 10 seconds later. And this is because they do some work when are Up, for instance standing up the DB

Reply:
As talked in a conversation we'll consider these case in an other ticket